docs/requirements.txt

```diff
@@ -0,0 +1,2 @@
+doxygen
+graphviz
```

Comment:
I think we also need to install graphviz on the github actions machine to get this working, see [this](https://pypi.org/project/graphviz/#:~:text=To%20render%20the%20generated%20DOT%20source%20code%2C%20you%20also%20need%20to%20install%20Graphviz%20(download%20page%2C%20archived%20versions%2C%20installation%20procedure%20for%20Windows).)

Reply:
Done. Have left the requirements.txt file in for now, see what happens...